code/font-awesome-4.7.0/HELP-US-OUT.txt

```diff
@@ -0,0 +1,7 @@
+I hope you love Font Awesome. If you've found it useful, please do me a favor and check out my latest project,
+Fort Awesome (https://fortawesome.com). It makes it easy to put the perfect icons on your website. Choose from our awesome,
+comprehensive icon sets or copy and paste your own.
+
+Please. Check it out.
+
+-Dave Gandy
```

Comment:
This **code/font-awesome** seems to take up a lot of space/files. Is this a package that is needed for the font to work? I´m new at this so really don´t know myself how it usually should look but it takes up 9000 lines of code and that don´t looks right.